workflow-templates/run-nextstrain-build-on-aws-batch.yaml

```diff
@@ -0,0 +1,28 @@
+name: Run Nextstrain build on AWS Batch
+
+on:
+  workflow_dispatch:
+    inputs:
+      image:
+        description: "Specific starter image to use for the build. Will override the default image for `nextstrain build`."
+        required: false
+        type: string
+
+jobs:
+  deploy-build-to-aws-batch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: nextstrain/.github/actions/run-nextstrain-build-on-aws-batch@master
+        with:
+          runtime-envvar-names: "GITHUB_RUN_ID PAT_GITHUB_DISPATCH SLACK_CHANNELS SLACK_TOKEN"
+        env:
+          NEXTSTRAIN_DOCKER_IMAGE: ${{ inputs.image }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          # Not auto-forwarded to builds, variable names must be passed via runtime-envvar-names
+          GITHUB_RUN_ID: ${{ github.run_id }}
```

Comment:
_non-blocking_

Why `GITHUB_RUN_ID`?

Reply:
In some of our existing builds, we have an `onstart` block that sends a Slack message that links to the GitHub Action run that started the build. These use the `GITHUB_RUN_ID` to construct the URL (e.g. [ncov-ingest/bin/notify-on-job-start)](https://github.com/nextstrain/ncov-ingest/blob/master/bin/notify-on-job-start)

Reply:
Ah. Hmm. That's a pretty distant coupling. I guess this is only a workflow template, but it seems likely to get copied/pasted as The New Incantation (similar to The Existing Incantation).

Ideally, in time, we have a better separation of concerns and move the notification stuff _out_ of the pathogen workflows themselves and into the automation (i.e. this action/workflow).